corehq/apps/es/es_query.py

```diff
@@ -466,6 +466,12 @@ def values_list(self, *fields, **kwargs):
     def count(self):
         return self.adapter.count(self.raw_query)
 
+    def exists(self):
```

Comment:
I was surprised to see ES didn't seem to have any better way of doing this.  From what I've read, this should be faster than a count, since it can terminate after the first result on each shard.